### PR TITLE
Budget Goggles post

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by amitaibu** [comment]
(post by @RachelBaram)

**Comment by RachelBaram** [comment]
@amitaibu Ready for review. I kept it very neutral.

File: _config.yml [hunk 1]
```diff
@@ -190,7 +190,7 @@ authors:
       bio: Somthing about Rachel
```
Comment (amitaibu) [inline _config.yml]: can you update this bio (one liner) as well please

File: app/_posts/2016-03-17-budget-goggles.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ The awareness of time increases productivity. Deliverables and deadline expectat
...
 
-In the case of lean demonstrations of a product idea (__Proof-of-Concept, Prototype, Minimal Viable Product__) we begin working with you before the scope is even defined. The uncertainty and inherent risk is immediately addressed by first timeboxing at the project-wide level. Web based product ideas can be successfully demonstrated within 3-4 months of development and with 1-3 developers.  This is easily translated into a budget range of $100K-$250K ($150K sweet spot). With this anchor, you can move forward confidently with your project.
+In the case of lean demonstrations of a product idea (__Proof-of-Concept, Prototype, Minimal Viable Product__) we begin working with our clients before the scope is even defined. The uncertainty and inherent risk is immediately addressed by first timeboxing at the project-wide level. Web based product ideas can be successfully demonstrated within 3-4 months of development and with 1-3 developers. This is easily translated into a budget range of $100K-$250K ($150K sweet spot). With this anchor, our clients can move forward confidently with the project.
```
Comment (amitaibu) [inline app/_posts/2016-03-17-budget-goggles.md]: @RachelBaram can you adapt also this "we begin", to propose people how to do it.

Also the `$100K-$250K ($150K sweet spot)` part should be maybe wrapped with, "for example in Gizra this means"

File: app/_posts/2016-03-17-budget-goggles.md [hunk 3]
```diff
@@ -11,9 +11,9 @@ author: RachelBaram
...
+Gizra addresses this by defining the ambiguous with the simple but powerful principle that the __budget dictates the project__. It all begins (and ends) at the price estimation. Our budget breakdown creates a common language for all stakeholders and sets expectations early on.
 
-After a __discovery phase__, your project is broken down to __timeboxed tasks__ - each with a clear deadline and cost to produce the deliverable. We set the maximum time to complete a task at 12 hours (cost of $1440) because this number is easily grasped by all stakeholders. Every feature and bug has a price tag and we encourage you _not_ to try to buy them all (at least not at first).
+After a __discovery phase__, the project is broken down to __timeboxed tasks__ - each with a clear deadline and cost to produce the deliverable. We set the maximum time to complete a task at 12 hours (cost of $1440) because this number is easily grasped by all stakeholders. Every feature and bug has a price tag and we encourage our clients _not_ to try to buy them all (at least not at first).
```
Comment (amitaibu) [inline app/_posts/2016-03-17-budget-goggles.md]: maybe change to `12 hours (for us it means a cost of $1440)`

**Comment by amitaibu** [comment]
I've removed the MVP price - felt a bit too "salesy" for a community blog post.

**Comment by amitaibu** [comment]
Merged, and deployed - http://www.gizra.com/content/budget-goggles/

@RachelBaram Mazal tov for your first Gizra post :)

**Pull request closed without merging**